HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -94,8 +188,10 @@ void computeWidth(const reco::HGCalMultiCluster& cluster, math::XYZPoint & bar,
 
 
 // ---------parameters ----------------------------
-bool readOfficialReco;
+//bool readOfficialReco;
```

Comment:
Please delete lines referring to `readOfficialReco` instead of commenting them out.

Reply:
Why getting rid of this flag and the associated possibility to read either simTracks/Vertices or TrackingParticles/Vertices? Can't we keep both?

Reply:
The tracking particles collection did not include all generated particles. That was the reason to switch.

Reply:
@rovere Do you see any advantage in keeping both collections or a use case for TrackingParticles/Vertices?

Reply:
maybe @malgeri can comment better on this topic.